src/Lean/Elab/BuiltinCommand.lean

```diff
@@ -138,6 +176,86 @@ private partial def elabChoiceAux (cmds : Array Syntax) (i : Nat) : CommandElabM
     aliases := aliases.push (currNamespace ++ id, declName)
   modify fun s => { s with env := aliases.foldl (init := s.env) fun env p => addAlias env p.1 p.2 }
 
+/-- Brings some definitions in scope.
```

Comment:
As I said somewhere else in this PR, I do think there is an issue of where the detailed discussion happens. To take the `namespace` example again, I would argue that the docstring should discuss the syntax and how it works, but not include a lengthy discussion of what a namespace is. Same for `universe`
One of the main arguments I see is that the manual must, I think, feature lengthy discussions about namespaces, universes and so on anyways. So personally I'd rather point to that in the docstring than maintaining two versions.

Reply:
As you said, if we go that route we should probably postpone this PR until we have something to point to in the manual

Reply:
I don't think we should postpone the PR for this, we should just merge the docs, and then later update things as our documentation system improves. The plan for the reference manual is to include docstrings for functions and syntax, so that we maintain a single canonical reference for everything, but this process will naturally result in reviewing and updating everything anyway.